manifests/a/Autodesk/DWGTrueView/24.2.72.0/Autodesk.DWGTrueView.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Autodesk.DWGTrueView
+PackageVersion: 24.2.72.0
+InstallerSwitches:
+  Silent: -suppresslaunch -d "C:\Temp\AutoDeskDWGTrueView"
+  SilentWithProgress: C:\Temp\AutoDeskDWGTrueView\Setup.exe -q

```

Comment:
SilentParameters still do not work. This application cannot be installed silently unless it is extracted first, which is not possible through winget